src/validator/loader.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+#==============================================================================
+# Copyright (C) 2017 Stephen F. Norledge and Alces Software Ltd.
+#
+# This file/package is part of Alces Metalware.
+#
+# Alces Metalware is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version.
+#
+# Alces Metalware is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this package.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For more information on the Alces Metalware, please visit:
+# https://github.com/alces-software/metalware
+#==============================================================================
+
+require 'file_path'
+require 'validator/answer'
+require 'validator/configure'
+require 'data'
+
+module Metalware
+  module Validator
+    class Loader
+      def initialize(metalware_config)
+        @config = metalware_config
+        @path = config.file_path
+      end
+
+      def configure
+        Validator::Configure.new(path.configure_file).load
+      end
+
+      def groups_cache
+        Data.load(path.groups_cache)
```

Comment:
Don't think this method needs to exist, we don't use it outside of tests (afaics) and its existence on the `Validator::Loader` class implies it will validate the loaded file, which it doesn't do.